### ISSUE TITLE
warp not working

### ISSUE THREAD

**Issue by smichaelsen** [Falkirks/SimpleWarp#55]
I'm new to PocketMine and this is the first plugin I'm trying so I beg for some patience :)

I installed SimpleWarp 3.3.7 and PocketMine-MP 1.7dev-743 implementing API version 3.0.0-ALPHA11 for Minecraft: PE v1.2.10.

The plugin is successfully loaded and the commands are available.
I was able to create a warp with `/addwarp test 256 166 256`.

Now I'm trying to warp a player to that warp: `/warp test playername` and this is what I get:

````
[21:42:39] [Server thread/INFO]: The target must hold still to complete the warp.
[21:42:44] [Server thread/CRITICAL]: Could not execute task falkirks\simplewarp\task\CommandWarpTask: Argument 2 passed to pocketmine\math\AxisAlignedBB::setBounds() must be of the type float, string given, called in phar:///etc/mcpe/sebastian/PocketMine-MP.phar/src/pocketmine/entity/Entity.php on line 627
[21:42:44] [Server thread/CRITICAL]: TypeError: "Argument 2 passed to pocketmine\math\AxisAlignedBB::setBounds() must be of the type float, string given, called in phar:///etc/mcpe/sebastian/PocketMine-MP.phar/src/pocketmine/entity/Entity.php on line 627" (EXCEPTION) in "src/pocketmine/math/AxisAlignedBB" at line 50
[21:42:44] [Server thread/DEBUG]: #0 src/pocketmine/entity/Entity(627): pocketmine\math\AxisAlignedBB->setBounds(double 255.7, string 166, double 255.7, double 256.3, double 167.8, double 256.3)
[21:42:44] [Server thread/DEBUG]: #1 src/pocketmine/entity/Entity(1733): pocketmine\entity\Entity->recalculateBoundingBox()
[21:42:44] [Server thread/DEBUG]: #2 src/pocketmine/entity/Entity(1749): pocketmine\entity\Entity->setPosition(pocketmine\level\Position object)
[21:42:44] [Server thread/DEBUG]: #3 src/pocketmine/entity/Entity(1842): pocketmine\entity\Entity->setPositionAndRotation(pocketmine\level\Position object, double 244.68537139893, double 35.171295166016)
[21:42:44] [Server thread/DEBUG]: #4 src/pocketmine/Player(3631): pocketmine\entity\Entity->teleport(pocketmine\level\Position object, NULL , NULL )
[21:42:44] [Server thread/DEBUG]: #5 SimpleWarp_v3.3.7.phar/src/falkirks/simplewarp/Destination(54): pocketmine\Player->teleport(falkirks\simplewarp\utils\WeakPosition object)
[21:42:44] [Server thread/DEBUG]: #6 SimpleWarp_v3.3.7.phar/src/falkirks/simplewarp/Warp(37): falkirks\simplewarp\Destination->teleport(pocketmine\Player object)
[21:42:44] [Server thread/DEBUG]: #7 SimpleWarp_v3.3.7.phar/src/falkirks/simplewarp/task/CommandWarpTask(45): falkirks\simplewarp\Warp->teleport(pocketmine\Player object)
[21:42:44] [Server thread/DEBUG]: #8 src/pocketmine/scheduler/TaskHandler(160): falkirks\simplewarp\task\CommandWarpTask->onRun(integer 21903)
[21:42:44] [Server thread/DEBUG]: #9 src/pocketmine/scheduler/ServerScheduler(326): pocketmine\scheduler\TaskHandler->run(integer 21903)
[21:42:44] [Server thread/DEBUG]: #10 src/pocketmine/Server(2540): pocketmine\scheduler\ServerScheduler->mainThreadHeartbeat(integer 21903)
[21:42:44] [Server thread/DEBUG]: #11 src/pocketmine/Server(2283): pocketmine\Server->tick()
[21:42:44] [Server thread/DEBUG]: #12 src/pocketmine/Server(2158): pocketmine\Server->tickProcessor()
[21:42:44] [Server thread/DEBUG]: #13 src/pocketmine/Server(1744): pocketmine\Server->start()
[21:42:44] [Server thread/DEBUG]: #14 src/pocketmine/PocketMine(556): pocketmine\Server->__construct(BaseClassLoader object, pocketmine\utils\MainLogger object, string /etc/mcpe/sebastian/, string /etc/mcpe/sebastian/plugins/)
[21:42:44] [Server thread/DEBUG]: #15 /etc/mcpe/sebastian/PocketMine-MP.phar(1): require(string phar:///etc/mcpe/sebastian/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
````

The player did not move at all. And when I finally move the player ingame:

````
[21:42:52] [Server thread/WARNING]: playername moved too fast, reverting movement
````

Any idea what I'm doing wrong?

**Comment by smichaelsen** [comment]
````
warpreport                                                  
sh: 1: xdg-open: not found
[21:53:33] [Server thread/INFO]: --- SimpleWarp Data ---
[21:53:33] [Server thread/INFO]: SERVER VERSION: 1.7dev-743
[21:53:33] [Server thread/INFO]: API: 3.0.0-ALPHA11
[21:53:33] [Server thread/INFO]: MCPE VERSION: v1.2.10
[21:53:33] [Server thread/INFO]: SOFTWARE: PocketMine-MP
[21:53:33] [Server thread/INFO]: SimpleWarp Version: 3.3.7
[21:53:33] [Server thread/INFO]: PLUGINS: Coords,SimpleWarp
[21:53:33] [Server thread/INFO]: storage-mode: 1
[21:53:33] [Server thread/INFO]: essentials-support: false
[21:53:33] [Server thread/INFO]: {
[21:53:33] [Server thread/INFO]:     "test": {
[21:53:33] [Server thread/INFO]:         "dest": "(X: 256, Y: 166, Z: 256, LEVEL: world)",
[21:53:33] [Server thread/INFO]:         "isPublic": false,
[21:53:33] [Server thread/INFO]:         "metadata": []
[21:53:33] [Server thread/INFO]:     }
[21:53:33] [Server thread/INFO]: }
````

**Comment by Falkirks** [comment]
This is fixed in 3b796721b7f059ec9e80658505b93943a831c816. Thank you for reporting it and providing the full error information. I will release version `3.3.8` with the fix shortly.
